src/components/Disputes/PrecedenceCampaign/PrecedenceCampaignBanner.js

```diff
@@ -0,0 +1,74 @@
+import React, { useCallback, useState } from 'react'
+import { Button, GU, useTheme } from '@aragon/ui'
+import Modal from './PrecedenceCampaignModal'
+
+const KEY_PREFIX = 'PC_DISPUTE_BANNER_READ'
+
+function PrecedenceCampaignBanner({ disputeId }) {
+  const theme = useTheme()
+  const [bannerVisible, setBannerVisible] = useState(
+    localStorage.getItem(`${KEY_PREFIX}${disputeId}`) === null
+  )
+  const [modalOpened, setModalOpened] = useState(false)
+
+  const closeBanner = useCallback(() => {
+    localStorage.setItem(`${KEY_PREFIX}${disputeId}`, 'true')
+    setBannerVisible(false)
+  }, [disputeId])
+
+  const openModal = useCallback(() => setModalOpened(true), [])
+  const markRead = useCallback(() => {
+    closeBanner()
+    setModalOpened(false)
+  }, [closeBanner, setModalOpened])
```

Comment:
No need to have the `setModalOpened` in the dependencies here :).

Reply:
Oops